src/components/organisms/NavBarSchedule/utils.ts

```diff
@@ -16,10 +16,19 @@ import { isTruthy } from "utils/types";
 
 export const prepareForSchedule = (
   day: Date,
-  usersEvents: MyPersonalizedSchedule
+  usersEvents: MyPersonalizedSchedule,
+  target: string
 ) => (event: WithVenueId<VenueEvent>): PersonalizedVenueEvent => {
```

Comment:
Since we have so many parameters here now, it would be good to use an object with named params rather than these 3 positional params.

The change will end up something like:

```typescript
export interface PrepareForScheduleProps {
  day: Date;
  usersEvents: MyPersonalizedSchedule;
  isForCalendarFile: boolean;
}

export const prepareForSchedule = ({
  day,
  usersEvents,
  isForCalendarFile,
}: PrepareForScheduleProps) => (event: WithVenueId<VenueEvent>): PersonalizedVenueEvent => {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1423/commits/ec68e834fab42e9549a285d7a454e4c29e1f24e4